tck/src/main/java/ee/jakarta/tck/concurrent/spec/Platform/dd/DeploymentDescriptorServlet.java

```diff
@@ -52,6 +59,120 @@ public class DeploymentDescriptorServlet extends TestServlet {
+                () -> assertNull(notInjectedContextD,
+                        "A contextService was injected with a qualifier which was not defined in it's deployment description"));
+
+        ManagedExecutorService lookupDefMES = InitialContext.doLookup("java:comp/DefaultManagedExecutorService");
+        ManagedExecutorService lookupMESD = InitialContext.doLookup("java:app/concurrent/ExecutorD");
+        
+        assertAll("Managed Executor Service",
+            () -> assertNotNull(injectedDefMES,
+                    "Default managedExecutorService was not injected when no qualifiers were present."),
+            () -> assertEquals(lookupDefMES, injectedDefMES,
+                    "Default managedExecutorService from injection was not the same as from lookup"),
+            () -> assertNotNull(injectedMESD,
+                    "Deployment Descriptor defined managedExecutorService was not inject with valid qualifier."),
+            () -> assertEquals(lookupMESD, injectedMESD,
+                    "The managedExecutorService from injection was not the same as from lookup"),
+            () -> assertNull(notInjectedMESD,
+                    "A managedExecutorService was injected with both a valid and invalid qualifier.")
+        );
+        
+        ManagedExecutorService lookupDefMSES = InitialContext.doLookup("java:comp/DefaultManagedScheduledExecutorService");
+        ManagedExecutorService lookupMSESD = InitialContext.doLookup("java:global/concurrent/ScheduledExecutorD");
+
+        assertAll("Managed Scheduled Executor Service",
+                () -> assertNotNull(injectedDefMSES,
+                        "Default managedScheduledExecutorService was not injected when no qualifiers were present."),
+                () -> assertEquals(lookupDefMSES, injectedDefMSES,
+                        "Default managedScheduledExecutorService from injection was not the same as from lookup"),
+                () -> assertNotNull(injectedMSESD,
+                        "Deployment Descriptor defined managedScheduledExecutorService was not inject with valid qualifiers."),
+                () -> assertEquals(lookupMSESD, injectedMSESD,
+                        "The managedScheduledExecutorService from injection was not the same as from lookup"),
+                () -> assertNull(notInjectedMSESD,
+                        "A managedScheduledExecutorService was injected with one of two required qualifiers.")
+        );
+        
+        ManagedThreadFactory lookupDefMTF = InitialContext.doLookup("java:comp/DefaultManagedThreadFactory");
+        ManagedThreadFactory lookupMTFD = InitialContext.doLookup("java:app/concurrent/ThreadFactoryD");
+
+        assertAll("Thread Factory",
+            () -> assertNotNull(injectedDefMTF,
+                    "Default managedThreadFactory was not injected when no qualifiers were present."),
+            () -> assertEquals(lookupDefMTF, injectedDefMTF,
```

Comment:
For ManagedThreadFactory, I think the specification actually requires a different instance per lookup because it says that context is captured at lookup time.